thirdparty/github.go

```diff
@@ -358,7 +362,7 @@ func getInstallationToken(ctx context.Context, owner, repo string, opts *github.
 	return token, nil
 }
 
-// revokeInstallationToken revokes an installation token.
+// RevokeInstallationToken revokes an installation token.
```

Comment:
Should we also make the comment note here about being careful about usage for this? If I'm reading this right it seems like it only still exists for dynamic github access token usage.